### ISSUE TITLE
table comments missing cut statement

### ISSUE THREAD

**Issue by mario-minati** [reneeb/MySQL-Workbench-DBIC#10]
Table comments with the new comment functionallity are generated without a closing `=cut` statement:

```
package Glue::DB::Schema::Core::Result::Contract;

# ABSTRACT: Result class for contract

=head1 DESCRIPTION

holds project contracts

use strict;
use warnings;
use base qw(DBIx::Class);
```

**Comment by reneeb** [comment]
Fixed in 1.12

**Comment by mario-minati** [comment]
Thanks for the quick fix 👍
